io/net/htx_nic_devices.py.data/htx_nic_devices.yaml

```diff
@@ -0,0 +1,9 @@
+mdt_file: 'net.mdt'
+host_ip: x.xx.xxx.xx
+peer_ip: x.xx.xxx.xy
+peer_user: "root"
+peer_password: "******"
```

Comment:
I need to check how feasible to is. Otherwise we will end up in doing lot of code changes.  I know password less ssh means there is no password, Just simply asking Is there any default password if we setup the password less ssh?